pkg/resolution/resolver/cluster/resolver_test.go

```diff
@@ -321,7 +321,7 @@ func TestResolve(t *testing.T) {
 					RefSource: &pipelinev1.RefSource{
 						URI: "/apis/tekton.dev/v1/namespaces/task-ns/task/example-task@a123",
 						Digest: map[string]string{
-							"sha256": sha256CheckSum(taskSpec),
```

Comment:
the sha256CheckSum is not used and can be removed.

Reply:
Agreed! Thanks

Reply:
Fixed. PTAL again 🙏